admin/proc-rhdh-deploy-eks-using-operator.adoc

```diff
@@ -113,7 +113,7 @@ You can include the required secret name in the `deployment.spec.template.spec.i
 
 [source]
 ----
-$ kubectl -n rhdh-operator patch deployment \
+kubectl -n rhdh-operator patch deployment \
     rhdh.fast --patch '{"spec":{"template":{"spec":{"imagePullSecrets":[{"name":"rhdh-pull-secret"}]}}}}' \
```

Comment:
```suggestion
    rhdh-operator --patch '{"spec":{"template":{"spec":{"imagePullSecrets":[{"name":"rhdh-pull-secret"}]}}}}' \
```

While we're at it. I just re-tested this on EKS and the Deployment name needs to be updated.